### PR TITLE
Add Heading rule to use Verb form to Style Guide

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

### Change Description ###

- We decided to use the verb form in the headings instead of gerund form.
  I add it to the Style Guide.
- (minor) Giving heading style to capitalization rule.

### CODE REVIEW

File: styleguide/style.md [hunk 1]
```diff
@@ -120,10 +120,25 @@ Lack of spacing makes it a little harder to read in source:
 Text after... DO NOT DO THIS.
 ```
 
+### Capitalization
+
 Use sentence-style capitalization. Always capitalize:
 - The first word of a heading.
 - The word following a colon in a title or heading (for example, "How to: Sort an array").
 
+### Verb form
```
Comment (journey2w) [inline styleguide/style.md:129]: I think you don't need to create a new section.
You can explain it in a new paragraph.

Comment (mijin85cho) [reply styleguide/style.md:129]: Thanks for the good reference, 
but aren't "starting sentence with verb form" and "capitalization" different issues?

I hoped to add "use verb for the heading",
but your reference seems to enhancing capitalization section.

Comment (journey2w) [reply styleguide/style.md:129]: The reference is about "title and heading capitalization". I want you to update this paragraph as above.
Authors can use not only verb phrase but also noun phrase.
It's okay. I will update this section later.

Comment (mijin85cho) [reply styleguide/style.md:129]: > The reference is about "title and heading capitalization". I want you to update this paragraph as above.

Does this comment mean to create a new PR to update the "### Capitalization" section like below?

### Capitalization

Use sentence-style capitalization. Always capitalize:
- The first word of a heading.
- The word following a colon in a title or heading (for example, "How to: Sort an array").

```markdown
Capitalize first word and proper nouns
```

```markdown
Capitalize First Word and Proper Nouns...DO NOT DO THIS
```

If so, I'll create a new PR.

**Pull request merged**